examples/lighting-app/efr32/BUILD.gn

```diff
@@ -45,6 +45,9 @@ declare_args() {
   # Monitor & log memory usage at runtime.
   enable_heap_monitoring = false
 
+  # Enable Sleepy end device
+  sleepy_device = false
```

Comment:
`args="sleepy_device=true"` does not seem as clear. Since the comment says enable, how about not commenting and saying `enable_sleepy_device` or `build_as_sleepy_device`.

Is this a common option? would it make sense to add it as a build variant to build_examples.py?

Reply:
yes enable_sleepy_device or enable_openthread_sed would have been clearer

Reply:
Done, it is clearer this way with enable_sleepy_device.

I think it is worth adding it to build_examples. I will do it shortly. I also have to do i for the new WiFI variant